src/lmnt/api.py

```diff
@@ -38,23 +41,49 @@ def __init__(self, status, error):
   def __str__(self):
     return f'SpeechError [status={self.status}] {self.message}'
 
+class StreamingSynthesisDoubleIterator:
```

Comment:
I am thinking naming it for what it's doing (iterating over message pairs to allow producing json-dict + audio-binary-data) rather than how it's used/instantiated (as a nested/double-layer iterator of sorts) may be clearer for new eng reading the code.

`StreamingSynthesisExtraData` or `MessagePairs`?

Also this is not intended to be a part of the public API, it is for internal use, maybe we should name it with an underscore prefix.